tests/pages/firefox/enterprise/landing.py

```diff
@@ -12,7 +12,22 @@ class EnterprisePage(FirefoxBasePage):
     URL_TEMPLATE = '/{locale}/firefox/enterprise/'
 
     _primary_download_button_locator = (By.ID, 'primary-download-button')
+    _win64_download_list_locator = (By.CSS_SELECTOR, '#win64-download-list.is-details')
```

Comment:
There's a menu list region that we can import here, to facilitate clicking and opening the menus:

https://github.com/mozilla/bedrock/blob/master/tests/pages/firefox/home.py#L8

https://github.com/mozilla/bedrock/blob/master/tests/pages/firefox/home.py#L25-L28